tensorboard/plugins/mesh/summary_v2.py

```diff
@@ -87,22 +76,22 @@ def _get_json_config(config_dict):
   return json_config
 
 
-def mesh(name, vertices, faces=None, colors=None, display_name=None,
-       description=None, config_dict=None, step=None):
+def mesh(name, vertices, faces=None, colors=None, description=None,
+    config_dict=None, step=None):
   """Writes a TensorFlow mesh summary.
 
   Args:
-    name: A name for this summary operation.
+    name: Uniquely identifiable name of the summary op. Could be replaced by
```

Comment:
Mind using the exact phrasing that the other summary ops use?  I think this is best for consistency.

```
name: A name for this summary. The summary tag used for TensorBoard will
      be this name prefixed by any active name scopes.
```